test/pylib/util.py

```diff
@@ -246,3 +247,11 @@ async def view_is_built():
         return done[0][0] == node_count or None
     deadline = time.time() + timeout
     await wait_for(view_is_built, deadline)
+
+
+async def wait_for_first_completed(coros: list[Coroutine]):
```

Comment:
If I understand correctly, the idea behind Patryk's previous comment was that such function already exists in existing test: `test_coordinator_queue_management.py`

You don't want to duplicate the code. Instead, you want to **move** the code.

So in addition to adding the function here in test/pylib/util.py, you should also remove it from test_coordinator_queue_management.py and instead `import` it from in that test.